test/cpp/qps/client.h

```diff
@@ -79,7 +83,7 @@ class Client {
 
   ClientStats Mark() {
     Histogram latencies;
-    std::vector<Histogram> to_merge(threads_.size());
+    Histogram to_merge[threads_.size()];  // avoid std::vector for old compilers
```

Comment:
I thought it looked unkosher but was just happy that it worked. I guess all our compilers support C99 extensions. That said, I need to back it out, obviously.